gdx/src/com/badlogic/gdx/Graphics.java

```diff
@@ -216,6 +216,8 @@ public String toString () {
 	 * android.util.DisplayMetrics#density, where one DIP is one pixel on an approximately 160 dpi screen. Thus on a 160dpi screen
 	 * this density value will be 1; on a 120 dpi screen it would be .75; etc.
 	 *
+	 * If no pixels per inch could be determined, this returns a default value of 1.
```

Comment:
Only minor critique I have: the term "pixels per inch" gives a subtle clue to how this works in AbstractGraphics, but the other implementing classes don't always use PPI to determine density (especially Android). Technically, this could be `If the density could not be determined, this returns a default value of 1.` but I have no major issue with how this is currently phrased. I don't think Android ever has a case where it can't determine density or PPI (although I could easily be wrong if emulators don't handle their display correctly), so the exact phrasing shouldn't matter.

Reply:
@tommyettinger Totally agree, changed it.